airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -17,24 +17,19 @@ definitions:
     url_base: "https://a.klaviyo.com/api/"
     authenticator: "#/definitions/authenticator"
     http_method: GET
-    error_handler:
-      type: CompositeErrorHandler
-      error_handlers:
-        - type: DefaultErrorHandler
-          backoff_strategies:
-            - type: WaitTimeFromHeader
-              header: "Retry-After"
-          response_filters:
-            - type: HttpResponseFilter
-              action: RETRY
-              http_codes: [429]
-        - type: DefaultErrorHandler # adding this DefaultErrorHandler for 5XX error codes
-        - type: DefaultErrorHandler
-          response_filters:
-            - type: HttpResponseFilter
-              action: FAIL
-              http_codes: [401, 403]
-              error_message: Please provide a valid API key and make sure it has permissions to read specified streams.
+    error_handlers:
+      type: DefaultErrorHandler
+      backoff_strategies:
+        - type: WaitTimeFromHeader
+          header: "Retry-After"
+      response_filters:
+        - type: HttpResponseFilter
+          action: RETRY
+          http_codes: [429]
+        - type: HttpResponseFilter
+          action: FAIL
```

Comment:
Should we treat this as a `config_error`?

Reply:
@artem1205 It should be `transient,` I believe since it's not actionable and it's a rate-limiting error. WDYT?

Reply:
Updated the 429 to be RATE_LIMITED

Updated the 401/403 to be a config_error.